seqr/views/utils/dataset_utils.py

```diff
@@ -192,8 +190,10 @@ def validate_variant_call_inputs(sample_type, dataset_path, project, elasticsear
     if dataset_path:
         if not dataset_path.endswith(".vcf.gz") and not dataset_path.endswith(".vds"):
             raise Exception("Variant call dataset path must end with .vcf.gz or .vds")
-        _validate_dataset_path(dataset_path)
-        _validate_vcf(dataset_path)
+
+        # TODO need to fix credentials
```

Comment:
If you don't plan to fix this in the very near future, could you make a github issue so we don't forget about this?

Reply:
good call.
https://github.com/macarthur-lab/seqr/issues/478